application/pom.xml

```diff
@@ -78,5 +78,18 @@
       <groupId>io.dropwizard</groupId>
       <artifactId>dropwizard-auth</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.restfb</groupId>
+      <artifactId>restfb</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sanction.lightning</groupId>
+      <artifactId>api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
You don't need the Guava dependency here. I think you meant to put it in `api.pom`